src/Manager.java

```diff
@@ -0,0 +1,31 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class Manager extends Employee{
```

Comment:
space after `Employee`, It is good habit to press shortcut for reformat code before pushing.

Reply:
Fixed. Will do it.